documentation/src/main/asciidoc/topics/ref_discovery_jdbc.adoc

```diff
@@ -1,6 +1,8 @@
+[id='discovery_jdbcping-{context}']
 = JDBC_PING
-`JDBC_PING` uses JDBC connections to shared databases, such as Amazon RDS on
-EC2, to store information about {brandname} nodes.
+JDBC_PING uses shared databases, such as Amazon RDS on EC2, to store information about {brandname} clusters.
```

Comment:
I think it works with "all" JDBC databases. Worth mention it? 
Also, link to docs: http://jgroups.org/manual4/index.html#_jdbc_ping